falcon/testing/client.py

```diff
@@ -2133,8 +2158,166 @@ async def __aexit__(self, exc_type, exc, tb):
         await self._task_req
 
 
+def _prepare_data_fields(data):
```

Comment:
ok this is the same as the first part of _encode_files at https://github.com/psf/requests/blob/15585909c3dd3014e4083961c8a404709450151c/requests/models.py#L155-L171